data/webbrowser.yaml

```diff
@@ -142,6 +142,207 @@ supported_os: [Windows,Darwin,Linux]
 labels: [Browser]
 urls: ['https://artifacts-kb.readthedocs.io/en/latest/sources/webbrowser/ChromeCache.html']
 ---
+name: ChromeAndDerivedBrowsersCache
```

Comment:
Isn't Chromium the base https://en.wikipedia.org/wiki/Chromium_(web_browser)

ChromeAndDerivedBrowsersCache => ChromiumBasedBrowsersCache

Reply:
Good catch, I didn't know the distinction between Chrome and Chromium. I'll update the descriptions so it doesn't say Chrome-based but rather Chromium-based browsers.